.github/workflows/call-release.yml

```diff
@@ -1,19 +1,27 @@
-# This workflow will build a Java project with Maven
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
-
 name: Release to github packages
 
 on:
   release:
     types: [created]
 
 jobs:
-  call-workflow-release:
-    uses: felleslosninger/eid-github-workflows/.github/workflows/maven-release-lib.yml@main
-    with:
-      java-version: 11
-    secrets:
-      packages-user: ${{ secrets.CREATE_PACKAGES_USER }}
-      packages-token: ${{ secrets.CREATE_PACKAGES_TOKEN }}
-      maven-user: ${{ secrets.MAVEN_USER }}
-      maven-password: ${{ secrets.MAVEN_PASSWORD }}
+  build:
```

Comment:
bruke lib her, uheldig med eigne workflows når me ønsker å gjere endringar... enklare med sentral lib for workflows